client/package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "BUILD_PATH=../server/public react-scripts build",
```

Comment:
[reference on build](https://stackoverflow.com/questions/43830866/what-is-npm-run-build-in-create-react-app)...simply saying, it is the same as npm start, but it will save loading time when running on both server and client-side when we want to see the features on both frontend and backend.